appveyor.yml

```diff
@@ -9,29 +9,37 @@ environment:
   matrix:
     # Note: 'default' toolchain is the same as 'vs-14-2015'
 
-    - TOOLCHAIN: "ninja-vs-12-2013-win64"
-      PROJECT_DIR: examples\foo
+#     - TOOLCHAIN: "ninja-vs-12-2013-win64"
+#       PROJECT_DIR: examples\foo
 
-    - TOOLCHAIN: "nmake-vs-12-2013-win64"
-      PROJECT_DIR: examples\foo
+#     - TOOLCHAIN: "nmake-vs-12-2013-win64"
+#       PROJECT_DIR: examples\foo
 
-    - TOOLCHAIN: "vs-12-2013-win64"
-      PROJECT_DIR: examples\foo
+#     - TOOLCHAIN: "vs-12-2013-win64"
+#       PROJECT_DIR: examples\foo
 
-    - TOOLCHAIN: "vs-12-2013-xp"
-      PROJECT_DIR: examples\foo
+#     - TOOLCHAIN: "vs-12-2013-xp"
+#       PROJECT_DIR: examples\foo
 
-    - TOOLCHAIN: "vs-12-2013"
-      PROJECT_DIR: examples\foo
+#     - TOOLCHAIN: "vs-12-2013"
+#       PROJECT_DIR: examples\foo
 
-    - TOOLCHAIN: "vs-14-2015"
-      PROJECT_DIR: examples\foo
+#     - TOOLCHAIN: "vs-14-2015"
+#       PROJECT_DIR: examples\foo
 
-    - TOOLCHAIN: "mingw"
-      PROJECT_DIR: examples\foo
+#     - TOOLCHAIN: "mingw"
+#       PROJECT_DIR: examples\foo
 
-    - TOOLCHAIN: "msys"
-      PROJECT_DIR: examples\foo
+#     - TOOLCHAIN: "msys"
+#       PROJECT_DIR: examples\foo
+
+      - TOOLCHAIN: "vs-15-2017-cxx17"
+        PROJECT_DIR: examples\FunctionalPlus
+
+      - TOOLCHAIN: "vs-15-2017-win64-cxx17"
+        PROJECT_DIR: examples\FunctionalPlus
+
+os: Visual Studio 2017
```

Comment:
Will `- APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017` do the trick?
* https://www.appveyor.com/docs/build-environment/#using-multiple-images-for-the-same-build